hack/deploy-and-e2e.sh

```diff
@@ -11,10 +11,22 @@ export NO_DOCKER=1
 export NAMESPACE=openshift-multiarch-tuning-operator
 oc create namespace ${NAMESPACE}
 oc annotate namespace ${NAMESPACE} \
-  scheduler.alpha.kubernetes.io/node-selector="kubernetes.io/arch=amd64" \
   workload.openshift.io/allowed="management"
 
 if [ "${USE_OLM:-}" == "true" ]; then
+  # Get the manifest from the manifest-list
+  # Prow produces a manifest-list image even for the bundle and that bundle image is not deployed on the multi-arch clusters
+  # Therefore, it can be a single-arch one and operator-sdk isn't able to extract the bundle as the bundle image is set in a
+  # pod's container image field and cri-o will fail to pull the image when the architecture of the node is different from the
+  # bundle image's architecture.
+  # However, the bundle image is FROM scratch and doesn't have any architecture-specific binaries. It doesn't need to be
+  # a manifest-list image. Therefore, we can extract the first single-arch manifest from the manifest-list image and use it
+  # as the bundle image in a multi-arch cluster, allowing the extraction pod to be scheduled on arm64 as well.
+  # The following is a workaround for this issue until https://issues.redhat.com/browse/DPTP-4143 is resolved.
+  if oc image info "${OO_BUNDLE}" | grep -q "Manifest List:"; then
```

Comment:
```suggestion
  if oc image info "${OO_BUNDLE}" --show-multiarch | grep -q "Manifest List:"; then
```

without the parameter `--show-multiarch`, the output is 
```
./oc image info "${OO_BUNDLE}"
error: the image is a manifest list and contains multiple images - use --filter-by-os to select from:

  OS          DIGEST
  linux/amd64 sha256:a21a7e87440a4f5321f23be317965b42057b485509d8b0948b7400def8d84343
  linux/arm64 sha256:7c84c5ff7591b34c6f984a09c261be39d2152621d02fc6c933413f7d5d822bf5
```